docs/src/components/header/HeaderNavDropdown.tsx

```diff
@@ -46,22 +46,22 @@ const UList = styled('ul')({
 const PRODUCTS = [
   {
     name: 'MUI Core',
-    description: 'Ready-to-use foundational components, free forever.',
+    description: 'Foundational components for shipping features faster.',
     href: ROUTES.productCore,
   },
   {
     name: 'MUI X',
-    description: 'Advanced and powerful components for complex use-cases.',
+    description: 'Advanced components for complex use cases.',
     href: ROUTES.productAdvanced,
   },
   {
     name: 'Templates',
-    description: 'Fully built, out-of-the-box, templates for your application.',
+    description: 'Professionally designed UI layouts to jumpstart your next project.',
```

Comment:
I think that we should stay in the terminology close to the one used in https://mui.com/store/ and https://mui.com/templates/. So I think that there are more changes needed on this one, at minimum to sync the messaging.

Reply:
I feel like the "curated by MUI" is an important part as it clearly states that it's not us who develop/design these.

Reply:
Agree that it's a message worth communicating. When we introduced "curated by MUI" in https://mui.com/store/, the motivation was to signal: only high-quality content. On the same topic of important words, I think that **template** is a big one. Also, I think that we should have something around starting a **new** project **faster**.